src/fides/api/ctl/database/crud.py

```diff
@@ -38,7 +38,7 @@ async def create_resource(
             already_exists_error = errors.AlreadyExistsError(
                 sql_model.__name__, resource_dict["fides_key"]
             )
-            log.bind(error=already_exists_error.detail["error"]).error(  # type: ignore[index]
+            log.bind(error=already_exists_error.detail["error"]).info(  # type: ignore[index]
```

Comment:
nice stealth change! I think we had an issue open for this

Reply:
Yes, after some comments I felt the need to tidy up the startup logs a bit. I'm surprised you didn't comment on the _other_ stealth logging change and massive QOL improvement in this PR... 😉